repairnator/pom.xml

```diff
@@ -61,7 +61,6 @@
         <module>repairnator-pipeline</module>
         <module>repairnator-core</module>
         <module>repairnator-realtime</module>
-        <module>repairnator-jenkins-plugin</module>
```

Comment:
why deleting?

Reply:
It makes the delivery phase after merging with master fails due to `401 code` per discussion (root try to stage in `Sonatype` and the does not allow `.hpi` I think) . I seems to manage to deploy this as a separate project , so maybe I should try to `cd` into `repairnator-jenkins-plugin` and `mvn deploy` after deploying the usual repairnator from root ?

Reply:
OK, we can keep it this way, but then we need to modify the travis config to also build this project, as we do for maven-repair

Reply:
Yes , It is done :) .